quint/src/graphics.ts

```diff
@@ -123,6 +121,99 @@ export function prettyQuintEx(ex: QuintEx): Doc {
   }
 }
 
+export function prettyQuintDef(
+  def: QuintDef,
+  includeBody: boolean = true,
+  type: TypeScheme | undefined = undefined
```

Comment:
```suggestion
  type?: TypeScheme
```

I think this is an equivalent but shorter parameter declaration :)